build_docs.py

```diff
@@ -590,7 +590,8 @@ def parse_args():
     parser.add_argument(
         "-b",
         "--branch",
-        metavar="3.6",
+        choices=dict.fromkeys(chain(*((v.branch_or_tag, v.name) for v in VERSIONS))),
```

Comment:
```suggestion
        choices=set(chain.from_iterable(((v.branch_or_tag, v.name) for v in VERSIONS))),
```
The struggle to make it sorted made it unreadable. Those choices are removed currently on main, but I were to reintroduce them, I'd do it as above.